packages/cf-component-checkbox/src/CheckboxGroup.js

```diff
@@ -1,8 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Checkbox from './Checkbox';
+import { applyTheme } from 'cf-style-container';
 import includes from 'lodash/includes';
 
+import CheckboxUnstyled from './Checkbox';
+import CheckboxTheme from './CheckboxTheme';
+
+const Checkbox = applyTheme(CheckboxUnstyled, CheckboxTheme, theme => ({
```

Comment:
Actually, I'm thinking about this and is this the pattern we want to do? Won't we just end up with a lot of props for every possible integration situation and a lot of complexity in the `Checkbox` component?